src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowParameters.java

```diff
@@ -23,6 +23,8 @@ public class DcLoadFlowParameters extends AbstractLoadFlowParameters {
 
     private final boolean distributedSlack;
 
+    private final boolean useActiveLimits;
```

Comment:
You don't need to have it here, as it is already in `LfNetworkParameters` which is embedded in DcLoadLowParameter (though the `AbstractLoadFlowParameters`)

Reply:
wow I did completely messed up here ... thanks